plugins/bap/utils/ida.py

```diff
@@ -102,23 +102,42 @@ def func_name_propagate_thunk(ea):
 def dump_brancher_info(output_filename):
     """Dump information for BAP's brancher into output_filename."""
     from idautils import CodeRefsFrom
-    import idc
 
     idaapi.autoWait()
 
-    def is_branch_insn(ea):
-        return len(list(CodeRefsFrom(ea, False))) > 0
-
-    def branch_list(ea):
+    def dest(ea):
         return list(CodeRefsFrom(ea, True))
 
+    def branch(ea):
+        return list(CodeRefsFrom(ea, False))
+
+    def fall(ea):
+        return list(set(dest(ea)) - set(branch(ea)))
+
+    def is_branch_insn(ea):  # Unconditional branches are also counted
+        return len(branch(ea)) > 0
+
+    def is_jump_insn(ea):  # Only unconditional branches
+        return len(branch(ea)) == len(dest(ea)) == 1
+
+    def labelled(addrs, label):
```

Comment:
Both forms of the word are valid English and mean the same. Different regions of the world use one more commonly over the other. But `labeled` is more terse :D